### PR TITLE
cleanign up previous commit

### LINKED ISSUE #48: Cleanup Recording.select_times()
@crheller: Your addition is very nice, but I would have asked for the following before accepting your pull request:

1) Since `Signal.select_times` returns a new signal, `Recording.select_times` should return a *new* Recording.
2) You accept a padding argument, but then silently discard it.
3) Always add unit tests for new functionality. 

Do you mind updating and submitting a new pull request? Be sure to include, in the message of the commit request, closes #48 and it will automatically close this issue when accepted.

### PR DESCRIPTION
**Pull request by crheller**
closes #48

### CODE REVIEW

File: nems/recording.py [hunk 1]
```diff
@@ -637,11 +637,16 @@ def select_epoch():
         raise NotImplementedError    # TODO
         
     def select_times(self, times, padding=0):
+        
+        if padding != 0:
+            raise NotImplementedError    # TODO
+        
         k = list(self.signals.keys())
+        newsigs = {}
         for sig in k:
```
Comment (bburan) [inline nems/recording.py]: You can also do:
```
for name, signal in self.signals.iteritems():
    newsigs[name] = signal.select_times(times)
```
Even more compact, using dictionary comprehension, is:
``newsigs = {name: signal.select_times(times) for name, signal in self.signals.iteritems()}
``
When using dict comprehension, it's fine to use very short variable names:
``newsigs = {n: s.select_times(times) for n, s in self.signals.iteritems()}
``

File: tests/test_recording.py [hunk 2]
```diff
@@ -1,8 +1,104 @@
...
+        'fs': fs,
+        'meta': {
+            'for_testing': True,
+            'date': "2018-01-10",
+            'animal': "Donkey Hotey",
+            'windmills': 'tilting'
+        },
+    }
+    return RasterizedSignal(**kwargs)
+
```
Comment (bburan) [inline tests/test_recording.py:80]: Minor nitpick. PEP8 requires two spaces between functions. Not that we've been 100% adhering to PEP8 ...

**Comment by bburan** [comment]
@crheller This looks great. Thanks for taking the time to put together the unit test. I know these can seem annoying at first, but they are invaluable when refactoring code. Just knowing that you have unit tests you can easily rewrite code and quickly know whether you broke something or not (and not 10 minutes later after the fitter throws an error when wrapping up).

Just some stylistic changes. Feel free to ignore my suggestions, but I do think list and dictionary comprehensions make code more readable.

**Comment by crheller** [comment]
@bburan I think I addressed both of those points now. Thanks!

**Comment by crheller** [comment]
@bburan don't accept yet, found a bug...

**Comment by crheller** [comment]
Should work now

**Review by bburan — APPROVED** [review]

**Comment by bburan** [comment]
Thanks!

**Pull request merged**